include/aspect/assembly.h

```diff
@@ -377,7 +377,7 @@ namespace aspect
          * - The copy object into which assemblers add up their contributions.
          */
         boost::signals2::signal<void (const double,
-                                      internal::Assembly::Scratch::StokesPreconditioner<dim>  &,
+                                      internal::Assembly::Scratch::StokesPreconditioner<dim> &,
```

Comment:
These changes are a bit of a shame since we had so nicely aligned the ampersands of arguments.

Could you limit the patch to only those files that really have functional changes?